git-hub

```diff
@@ -42,12 +42,12 @@ import sys
 import time
 import json
 import base64
-import urllib
+import urllib.request, urllib.parse, urllib.error
 import pprint
-import urllib2
+import urllib.request, urllib.error, urllib.parse
```

Comment:
I guess this should be removed as they are already imported above?

Reply:
Yes. My bad. Expected `2to3` to generate perfect code :laughing: